internal/stringify/fmt_test.go

```diff
@@ -30,7 +30,26 @@ func (a aConcealer) Conceal() string                { return "***" }
 func (a aConcealer) Format(fs fmt.State, verb rune) { io.WriteString(fs, "***") }
 func (a aConcealer) PlainString() string            { return fmt.Sprintf("%v", a.v) }
 
+type testPtrStruct struct {
+	ptrV1 *string
+	inner *testPtrInnerStruct
+}
+
+type testPtrInnerStruct struct {
+	ptrV2 *string
+}
+
 func TestFmt(t *testing.T) {
+	ptrTestString := "ptrString"
+	ptrString := &ptrTestString
+
+	testStruct := &testPtrStruct{
```

Comment:
this makes the purpose of this variable more obvious.
```suggestion
	structWithPointerFields := &testPtrStruct{
```